src/routes/index.ts

```diff
@@ -384,6 +385,15 @@ router.route(config.OVERSEAS_ENTITY_REVIEW_URL)
   .get(overseasEntityReview.get)
   .post(overseasEntityReview.post);
 
+router.route(config.BENEFICIAL_OWNER_BO_MO_REVIEW_URL)
+  .all(
+    authentication,
+    companyAuthentication,
+    navigation.hasUpdatePresenter
```

Comment:
Was this nav check suggested by product? It looks like there's an interrupt card just before this new page which might make more sense to check against but worth asking your BA

Reply:
The order has shifted on merge, so no interrupt card right before this page.